packages/core/src/util/postHog.ts

```diff
@@ -0,0 +1,16 @@
+import { PostHog } from "posthog-node";
```

Comment:
@psugihara do you know of a way to share this file across cli and core so I don't have to duplicate it like this?

Reply:
not sure what's best but right now we either
- duplicate files
- export from mailing-core (though i don't think this makes sense to export as a public api)

we could add a shared package or use a symlink

Reply:
Just gonna copy for now.  The main thing we need to decide is how we want to get the anonymousId into the sendMail call